test/mix/helpers_test.exs

```diff
@@ -84,4 +84,37 @@ defmodule Mix.Appsignal.HelperTest do
       assert Mix.Appsignal.Helper.agent_platform() == "freebsd"
     end
   end
+
+  describe ".check_proxy" do
+    test "check_proxy returns nil if there's no proxy defined" do
+      assert Mix.Appsignal.Helper.check_proxy(%{}) == nil
+    end
+
+    test "check_proxy uses APPSIGNAL_HTTP_PROXY / https_proxy / HTTPS_PROXY (in that order)" do
+      env = %{"HTTPS_PROXY" => "MY_HTTPS_PROXY"}
+
+      assert Mix.Appsignal.Helper.check_proxy(env) == {"HTTPS_PROXY", "MY_HTTPS_PROXY"}
+
+      env = Map.put(env, "https_proxy", "my_https_proxy")
+
+      assert Mix.Appsignal.Helper.check_proxy(env) == {"https_proxy", "my_https_proxy"}
+
+      env = Map.put(env, "APPSIGNAL_HTTP_PROXY", "my_appsignal_proxy")
+
+      assert Mix.Appsignal.Helper.check_proxy(env) ==
+               {"APPSIGNAL_HTTP_PROXY", "my_appsignal_proxy"}
+    end
+
+    test "check_proxy ignores http_proxy / HTTP_PROXY" do
```

Comment:
This fails in the CI. I don't see any logic for this behavior either. Do we want to ignore this env variable?

Reply:
Ahh well.. No.. That was before @jeffkreeftmeijer added back `http_proxy` support. With his change, the tests will need some updates too 😉

Reply:
Ah, missed that. I'll dig around in the code to find your original commit and push that.

Reply:
Alright, added https://github.com/appsignal/appsignal-elixir/pull/458/commits/4bc9bad64e75a61c5fe5f7d519e9096fe557db88 to test http_proxy and HTTP_PROXY as well.